packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -54,43 +60,45 @@ git clone git@github.com:department-of-veterans-affairs/vets-api.git
 git clone git@github.com:department-of-veterans-affairs/vets-api-mockdata.git
 ```
 
-These repos are the most important to have a basic understanding of:
-
-- vets-website
-  - Front end platform and application code
-- vagov-content
-  - Markdown content used to generate static pages (must be a sibling of vets-website)
-- vets-json-schema
-  - Shared JSON Schema definitions used by form applications and the APIs that they consume
-- veteran-facing-services-tools
-  - Shared code that we support for the broader VA (not just VA.gov), plus this documentation site
-- vets-api
-  - The repository for the Rails API backend.
-- vets-api-mockdata
-  - Mock data used when running locally and on dev for the backend (must be a sibling of vets-api)
+*Front end repos*
```

Comment:
I think we should move this to a separate document and link to it here but nothing currently in the repo makes sense- main reason is to cut down on the cruft when giving a list of instructions.